src/main/java/org/folio/rest/converter/CommonAttributesConverter.java

```diff
@@ -65,6 +61,7 @@ public Token convertToken(TokenInfo tokenInfo) {
   }
 
   public List<TitleContributors> convertContributors(List<org.folio.rmapi.model.Contributor> contributorList) {
+    if(Objects.isNull(contributorList)) return Collections.emptyList();
```

Comment:
According to Java convention every if statement should use curly braces to avoid confusion about what lines will be executed

Reply:
Ok, will adjust -- think the single line shorthand/ single line was suggested through IDE plugin (just recently switched to Intellij)